File: week-04/day-01/sharpie.py
class Sharpie(object):

    def __init__(self, color, width):
        self.ink_amount = 100
        self.color = color
        self.width = float(width)
    
    def use(self):
        self.ink_amount -= 1

filc_1 = Sharpie("red", 12)

filc_1.use()
filc_1.use()
print(filc_1.color, filc_1.width, filc_1.ink_amount)

File: week-08/day-01/sharpie.js
class Sharpie {

  constructor(color, width) {
    this.color = color;
    this.width = width;
    this.inkAmount = 100;
  }

  use() {
    this.inkAmount -= this.width;
  }
}

var sharpie1 = new Sharpie("red", 12);
var sharpie2 = new Sharpie("blue", 20);
sharpie1.use();
sharpie1.use();
console.log(sharpie1.width);
console.log(sharpie1.inkAmount);

console.log(sharpie2.inkAmount);
while (sharpie2.inkAmount > 0) {
  sharpie2.use();
  console.log(sharpie2.inkAmount);
}